### ISSUE TITLE
Strong name for Windows 10 library

### ISSUE THREAD

**Issue by csabar** [oxyplot/oxyplot#731]
I am getting an annoying warning about missing strong name in Windows 10 UAP library:
```
CSC : warning CS8002: Referenced assembly 'OxyPlot.Windows, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' does not have a strong name.
```

**Comment by GeertvanHorrik** [comment]
No need to sign, I can submit apps with OxyPlot to the store.

**Comment by csabar** [comment]
Maybe it is not needed, but it is the only warning in the project and this is annoying. Signing an assembly is an easy fix IMO.

**Comment by GeertvanHorrik** [comment]
adding 8002 to the ignored warnings is even easier ;-)

Signing a lib means managing the key, signing it in the build, etc. For what purpose?

**Comment by tibel** [comment]
The other OxyPlot assemblies have a strong name already.
Adding a strong name to the Win10 assemblies is the most obvious fix, I think.

**Comment by tibel** [comment]
fixed in `1.0.0-unstable1968`

**Comment by csabar** [comment]
Fixed
